server/link.go

```diff
@@ -359,7 +354,38 @@ func (me *SSHServer) handleAPIUnlink(s ssh.Session) {
 	me.config.Stats.APIUnlink()
 }
 
-func (me *SSHServer) sendLink(lt charm.LinkTransport, lc chan *charm.Link, l *charm.Link) {
+type sshLinkQueue struct {
```

Comment:
Maybe we should call `sshLinkQueue` `channelLinkQueue` or `localLinkQueue` since all linking will happen over ssh and that's a bit more descriptive of the concrete implementation of the interface.